### PR TITLE
Add Release Notes for Android 1.13.x Release

### LINKED ISSUE #11596: [Android] Release Notes for Android 1.13.x Release
- Implemented cross-platform promotion modal. ([#10571](https://github.com/brave/brave-browser/issues/10571))
- Implemented "Refresh Status" button on rewards panel to check for publisher status. ([#10005](https://github.com/brave/brave-browser/issues/10005))
- Implemented version 2 of onboarding. ([#9555](https://github.com/brave/brave-browser/issues/9555))
- Implemented fingerprint farbling for Plugins. ([#9435](https://github.com/brave/brave-browser/issues/9435))
- Implemented new wallet API for Brave Rewards. ([#8428](https://github.com/brave/brave-browser/issues/8428))
- Disabled ability to retrieve rewards logs via brave://rewards-internals and removed existing logs. ([#11260](https://github.com/brave/brave-browser/issues/11260))
- Fixed issue where ad conversions would fail when a site uses pushState. ([#11012](https://github.com/brave/brave-browser/issues/11012))
- Fixed crash issue when opening search settings. ([#10773](https://github.com/brave/brave-browser/issues/10773))
- Fixed an issue with opening downloaded PDF files. ([#11420](https://github.com/brave/brave-browser/issues/11420))
- Fixed sending links from Android to Desktop when sync is enabled. ([#11077](https://github.com/brave/brave-browser/issues/11077))
- [Security] Reduce logging as reported on HackerOne by hihouhou. ([#11456](https://github.com/brave/brave-browser/issues/11456))
- Upgraded Chromium to 85.0.4183.83. ([#11401](https://github.com/brave/brave-browser/issues/11401))

### PR DESCRIPTION
**Pull request by srirambv**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Test Plan:
Closes #11596

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: CHANGELOG_ANDROID.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+ - Implemented version 2 of onboarding. ([#9555](https://github.com/brave/brave-browser/issues/9555))
+ - Implemented fingerprint farbling for Plugins. ([#9435](https://github.com/brave/brave-browser/issues/9435))
+ - Implemented new wallet API for Brave Rewards. ([#8428](https://github.com/brave/brave-browser/issues/8428))
+ - Disabled ability to retrieve rewards logs via brave://rewards-internals and removed existing logs. ([#11260](https://github.com/brave/brave-browser/issues/11260))
+ - Fixed issue where ad conversions would fail when a site uses pushState. ([#11012](https://github.com/brave/brave-browser/issues/11012))
+ - Fixed crash issue when opening search settings. ([#10773](https://github.com/brave/brave-browser/issues/10773))
+ - Fixed an issue with opening downloaded PDF files. ([#11420](https://github.com/brave/brave-browser/issues/11420)
+ - Fixed sending links from Android to Desktop when sync is enabled. ([#11077](https://github.com/brave/brave-browser/issues/11077)
+ - [Security] Reduce logging as reported on HackerOne by hihouhou. ([#11456](https://github.com/brave/brave-browser/issues/11456))
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:15]: `[Security]` issues should always be near the top. Examples:

* https://github.com/brave/brave-browser/blob/master/CHANGELOG_DESKTOP.md#11382
* https://github.com/brave/brave-browser/blob/master/CHANGELOG_DESKTOP.md#112114

Can put this one below the last `Implemented` item.

File: CHANGELOG_ANDROID.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.13.85](https://github.com/brave/brave-browser/releases/tag/v1.13.85)
+
+ - Implemented cross-platform promotion modal. ([#10571](https://github.com/brave/brave-browser/issues/10571))
+ - Implemented "Refresh Status" button on rewards panel to check for publisher status. ([#10005](https://github.com/brave/brave-browser/issues/10005))
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:6]: Can remove `for` from the sentence. Reads better as:

* `Implemented "Refresh Status" button on rewards panel to check publisher status.`

File: CHANGELOG_ANDROID.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+ - Implemented version 2 of onboarding. ([#9555](https://github.com/brave/brave-browser/issues/9555))
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:7]: Probably can word this a bit better. User has no idea what "version 2" means. Maybe something along the lines of the following:

* `Implemented new version of onboarding`

File: CHANGELOG_ANDROID.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+ - Implemented fingerprint farbling for Plugins. ([#9435](https://github.com/brave/brave-browser/issues/9435))
+ - Implemented new wallet API for Brave Rewards. ([#8428](https://github.com/brave/brave-browser/issues/8428))
+ - Disabled ability to retrieve rewards logs via brave://rewards-internals and removed existing logs. ([#11260](https://github.com/brave/brave-browser/issues/11260))
+ - Fixed issue where ad conversions would fail when a site uses pushState. ([#11012](https://github.com/brave/brave-browser/issues/11012))
+ - Fixed crash issue when opening search settings. ([#10773](https://github.com/brave/brave-browser/issues/10773))
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:12]: Can probably remove `issue`. Example:

* `Fixed crash when opening search settings.`

BTW, was this on GPS? Sounds like it wasn't a regression as per https://github.com/brave/brave-browser/issues/10773. Should this be removed from the release notes?

File: CHANGELOG_ANDROID.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
+ - Fixed an issue with opening downloaded PDF files. ([#11420](https://github.com/brave/brave-browser/issues/11420)
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:13]: Maybe something along the lines of the following?

* `Fixed not being able to open downloaded PDF and APK files.`

File: CHANGELOG_ANDROID.md [hunk 6]
```diff
@@ -1,5 +1,20 @@
...
+ - Fixed sending links from Android to Desktop when sync is enabled. ([#11077](https://github.com/brave/brave-browser/issues/11077)
```
Comment (kjozwiak) [inline CHANGELOG_ANDROID.md:14]: Needs to describe what the issue was. Above doesn't describe what was broken.

* ` - Fixed not being able to send links from Android to Desktop when sync is enabled.`

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG_ANDROID.md (hunk 4 above)
Comment (srirambv) [reply CHANGELOG_ANDROID.md:12]: Maybe @samartnik  can confirm. I just added it as it was marked as include

Comment (samartnik) [reply CHANGELOG_ANDROID.md:13]: @srirambv  it was not in GPS, it was discovered and fixed in nightly only

**Pull request merged**